UNITTESTS/features/cellular/framework/AT/athandler/athandlertest.cpp

```diff
@@ -947,6 +947,32 @@ TEST_F(TestATHandler, test_ATHandler_read_int)
 
     ret = at.read_int();
     EXPECT_TRUE(2 == ret);
+    at.resp_stop();
```

Comment:
As these changes are not related to the PR and commit description, I just want to make sure that they are intended and did not get added by mistake?

Reply:
Good catch @michalpasztamobica. That slipped from other thing here. I will remove this

Reply:
Removed now